### ISSUE TITLE
DCAT feed generated multiple times

### ISSUE THREAD

**Issue by peterdesmet** [oSoc15/ipt-dcat#40]
Setup: a test IPT registered with a test organization. No published datasets.

The DCAT feed I get is this (note: I have hidden one URL):

```
@prefix adms: <http://www.w3.org/ns/adms#> .
@prefix schema: <http://schema.org/> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix dcat: <http://www.w3.org/ns/dcat#> .
@prefix foaf: <http://xmlns.com/foaf/0.1/> .
@prefix dct: <http://purl.org/dc/terms/> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix locn: <http://www.w3.org/ns/locn#> .
@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
@prefix vcard: <http://www.w3.org/2006/vcard/ns#> .

<http://.../ipt-dcat/#catalog>
 a dcat:Catalog ;
dct:title "IPT DCAT" ;
dct:description "IPT DCAT" ;
dct:publisher <http://www.gbif.org/publisher/b2069a2e-0fb3-4193-9fee-1910694cfca5/#Organization> ;
dct:issued "2015-07-29T09:19+02:00" ;
dct:modified "2015-07-29T09:19+02:00" ;
dcat:themeTaxonomy <http://eurovoc.europa.eu/218403> ;
dct:rights <https://creativecommons.org/publicdomain/zero/1.0/> .

<http://eurovoc.europa.eu/218403> a skos:ConceptScheme ; dct:title "biodiversity"@en .

<http://www.gbif.org/publisher/b2069a2e-0fb3-4193-9fee-1910694cfca5/#Organization> a foaf:Agent ; foaf:name "IPT DCAT" .
```

If I wait a couple of minutes and **reload**, I get this:

```
@prefix adms: <http://www.w3.org/ns/adms#> .
@prefix schema: <http://schema.org/> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix dcat: <http://www.w3.org/ns/dcat#> .
@prefix foaf: <http://xmlns.com/foaf/0.1/> .
@prefix dct: <http://purl.org/dc/terms/> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix locn: <http://www.w3.org/ns/locn#> .
@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
@prefix vcard: <http://www.w3.org/2006/vcard/ns#> .

<http://.../ipt-dcat/#catalog>
 a dcat:Catalog ;
dct:title "IPT DCAT" ;
dct:description "IPT DCAT" ;
dct:publisher <http://www.gbif.org/publisher/b2069a2e-0fb3-4193-9fee-1910694cfca5/#Organization> ;
dct:issued "2015-07-29T09:19+02:00" ;
dct:modified "2015-07-29T09:19+02:00" ;
dcat:themeTaxonomy <http://eurovoc.europa.eu/218403> ;
dct:rights <https://creativecommons.org/publicdomain/zero/1.0/> .

<http://eurovoc.europa.eu/218403> a skos:ConceptScheme ; dct:title "biodiversity"@en .

<http://www.gbif.org/publisher/b2069a2e-0fb3-4193-9fee-1910694cfca5/#Organization> a foaf:Agent ; foaf:name "IPT DCAT" .

@prefix adms: <http://www.w3.org/ns/adms#> .
@prefix schema: <http://schema.org/> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix dcat: <http://www.w3.org/ns/dcat#> .
@prefix foaf: <http://xmlns.com/foaf/0.1/> .
@prefix dct: <http://purl.org/dc/terms/> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix locn: <http://www.w3.org/ns/locn#> .
@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
@prefix vcard: <http://www.w3.org/2006/vcard/ns#> .

<http://.../ipt-dcat/#catalog>
 a dcat:Catalog ;
dct:title "IPT DCAT" ;
dct:description "IPT DCAT" ;
dct:publisher <http://www.gbif.org/publisher/b2069a2e-0fb3-4193-9fee-1910694cfca5/#Organization> ;
dct:issued "2015-07-29T09:49+02:00" ;
dct:modified "2015-07-29T09:49+02:00" ;
dcat:themeTaxonomy <http://eurovoc.europa.eu/218403> ;
dct:rights <https://creativecommons.org/publicdomain/zero/1.0/> .

<http://eurovoc.europa.eu/218403> a skos:ConceptScheme ; dct:title "biodiversity"@en .

<http://www.gbif.org/publisher/b2069a2e-0fb3-4193-9fee-1910694cfca5/#Organization> a foaf:Agent ; foaf:name "IPT DCAT" .
```

The information is repeated, only the issued/modified timestamps are different. If I wait and reload after that, the content gets repeated again. I don't think this is intentional.

**Comment by simon-vc** [comment]
We fixed this one
No idea why we didn't get it before

**Comment by peterdesmet** [comment]
Confirmed fixed in `v1.2`
